website/js/print.js

```diff
@@ -38,6 +38,7 @@ function print_selected() {
   
   window.open("render-document.php/" + doc_class_details['type'] + "/" + doc_class
               + "?options=" + encodeURIComponent(JSON.stringify(options))
+              + "&sort=" + encodeURIComponent($("#sortorder-racers option:selected").val())
```

Comment:
See below.  Ordering is determined entirely by the order in which the ids appear.

Reply:
See previous -- I reverted this change.